File: starters/CSharp/Bomberjam/Player.cs
using System.Text.Json.Serialization;

namespace MyBot.Bomberjam
{
    public class Player
    {
        [JsonPropertyName("id")]
        public string Id { get; set; }

        [JsonPropertyName("name")]
        public string Name { get; set; }

        [JsonPropertyName("x")]
        public int X { get; set; }

        [JsonPropertyName("y")]
        public int Y { get; set; }

        [JsonPropertyName("startingCorner")]
        public string StartingCorner { get; set; }

        [JsonPropertyName("bombsLeft")]
        public int BombsLeft { get; set; }

        [JsonPropertyName("maxBombs")]
        public int MaxBombs { get; set; }

        [JsonPropertyName("bombRange")]
        public int BombRange { get; set; }

        [JsonPropertyName("isAlive")]
        public bool IsAlive { get; set; }

        [JsonPropertyName("timedOut")]
        public bool IsTimedOut { get; set; }

        [JsonPropertyName("respawning")]
        public int Respawning { get; set; }

        [JsonPropertyName("score")]
        public int Score { get; set; }

        [JsonPropertyName("color")]
        public int Color { get; set; }
    }
}

File: starters/PHP/bomberjam/Player.php
<?php

namespace bomberjam;

class Player
{
    /** @var int */
    public $x;

    /** @var int */
    public $y;

    /** @var string */
    public $id;

    /** @var int */
    public $bombsLeft;

    /** @var int */
    public $maxBombs;

    /** @var int */
    public $bombRange;

    /** @var bool */
    public $isAlive;

    /** @var bool */
    public $timedOut;

    /** @var int */
    public $score;

    /** @var int */
    public $color;

    /** @var int */
    public $respawning;

    /** @var string */
    public $name;

    /** @var string */
    public $startingCorner;

    public function __construct($jsonPlayer)
    {
        $this->x = $jsonPlayer['x'];
        $this->y = $jsonPlayer['y'];
        $this->id = $jsonPlayer['id'];
        $this->bombsLeft = $jsonPlayer['bombsLeft'];
        $this->maxBombs = $jsonPlayer['maxBombs'];
        $this->bombRange = $jsonPlayer['bombRange'];
        $this->isAlive = $jsonPlayer['isAlive'];
        $this->timedOut = $jsonPlayer['timedOut'];
        $this->respawning = $jsonPlayer['respawning'];
        $this->score = $jsonPlayer['score'];
        $this->color = $jsonPlayer['color'];
        $this->name = $jsonPlayer['name'];
        $this->startingCorner = $jsonPlayer['startingCorner'];
    }
}